e2-studio-star-rx72n-firmware/libs/rx_spi_link/src/rx_spi_link.c

```
@@ -0,0 +1,731 @@
+    }
+  }
+
+  return k_rx_ok;
+}
+
+/**
+ * @brief Receive and decode a frame from SPI with HARQ
+ * @see rx_spi_link.h for full documentation
+ */
+rx_err_t
+rx_spi_link_receive(rx_spi_link_t* link, rx_spi_link_receive_result_t* result, uint32_t timeout_ms)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_spi_link_receive' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)